packages/dart_frog/lib/src/response.dart

```diff
@@ -95,7 +95,7 @@ class Response {
           encoding: encoding,
         );
 
-  /// A [shelf.Response.context] key used to determine if if the
```

Comment:
So I think this is actually a bug in the lint rule. `shelf` is included as part of an import and it doesn't seem to realize that. The simple fix is just removing the brackets so it just reads it as text instead of a object link.

Reply:
Gonna resolve this given the update below from Jochum